### DISCUSSION TITLE
New field type: DURATION

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#5490] [Ideas]
Postgres has an `interval` field type.
We will probably need to introduce something similar to display durations (e.g. 1 day 2 hours and 3 seconds).

**Comment by BOHEUS** [comment]
@FelixMalfait is it still relevant? One use case I see would be related to requested timeline view (though I think it'd be simpler to achieve this with 2 separate date fields rather than date and duration fields https://github.com/twentyhq/twenty/discussions/8696)
